apps/docs-snippets2/src/introduction/connecting-to-localnode.ts

```diff
@@ -0,0 +1,14 @@
+// #region main
+import { Provider, Wallet } from 'fuels';
+
+import { LOCAL_NETWORK_URL } from '../env';
+
+const ADDRESS = '0x767caf5b08eba21c561078a4d5be09bbd7f16b9eca22699a61f1edd9e456126f';
+
+const provider = await Provider.create(LOCAL_NETWORK_URL);
+const wallet = Wallet.fromAddress(ADDRESS, provider);
+
+const { balances } = await wallet.getBalances();
+
+console.log(`Balances should be an array`, balances instanceof Array);
```

Comment:
@arboleya I had actually thought the `console.assert` pattern was more expressive as it offered a pass/fail debug experience. What was the rationale for the change to `console.log` ?

Reply:
I saw no advantage in using `assert()` over `log()`, since both require the same [patching](https://github.com/FuelLabs/fuels-ts/blob/254bb621e2b9f4835cc726d7e394b15cf36ce25f/apps/docs-snippets2/scripts/test-template.ts#L20-L27). Also, native `log()` always logs something, while `assert()` logs only failures. Ultimately, going with `log()` also goes well with the initial idea of removing testing concepts from snippets.

In fact, I'd argue `log` is more _~expressive~_ natural (clean) for a code snippet. For example, this snippet you commented on would look much better like this while still passing the same `!!truthy` validation.

```ts
console.log(`Balances`, balances);
```


Issue added:
 - https://github.com/FuelLabs/fuels-ts/issues/3205

Reply:
Good points.